src/error_histogram_service.py

```diff
@@ -0,0 +1,183 @@
+import os.path
+import shutil
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import atlassian.errors
+import pandas as pd
+import matplotlib.colors as mc
+import matplotlib.pyplot as plt
+import numpy as np
+from src.common import ConfluenceConnection, ConfluenceNodeMapper
+
+
+class DataManager:
```

Comment:
Do I understand it correctly that this class downloads the CSV file attachements from confluence to compute the error histogram??? The CSVHandler/InfoCSVHandler are classes that were created exactly for this purpose...

Reply:
I removed the Data Manager class and now use a CSV reader to read the stat files from the directory directly